guides/using-helpers.md

```diff
@@ -4,4 +4,166 @@ layout: guides
 permalink: /guides/using-helpers/
 ---
 
-#Using Helpers
+# Using Helpers
+
+Helpers are extensions added to Dust to increase its functionality. They make it possible for anyone to extend the way Dust works. They are commonly used to add functionality related to things like logic, formatting, and internationalization. Helpers are denoted by the `@` symbol. Let's take a look at some commonly used helpers.
+
+## Installing Helpers
+
+Helpers are bundled separately as the `dustjs-helpers` library. After the dustjs-linkedin library is loaded, follow the same instructions in the [Getting Started guide](/guides/getting-started/) to install `dustjs-helpers`. If you are cloning or downloading from GitHub, you can find the helpers in the [dustjs-helpers repository](https://github.com/linkedin/dustjs-helpers).
+
+If you are using node.js, then the following `require` statements will ensure you have the needed Dust modules available.
+
+```
+require('dustjs-linkedin');
+require('dustjs-helpers');
+```
+
+## Logic Helpers
+
+The helpers library comes with the following logic helpers: `{@eq}`, `{@ne}`, `{@gt}`, `{@lt}`, `{@gte}`, and `{@lte}`. These helpers allow you to print content if one value compared in a certain way (e.g. equal, greater than or equal to) to another value is true. For each helper, you specify the first value with the `key` attribute and the second value with the `value` attribute. Both `key` and `value` can point to a reference or a literal value. Wrap literal values in quotes and leave references un-quoted.
```

Comment:
I would put the parentheses after "to another value":

`compared in a certain way to another value (e.g. equal, greater than or equal to) ...`